pages/landing/hipaa/technical-details/_how-it-works.html

```diff
@@ -32,7 +32,7 @@
             </div>
           {% endfor %}
         </div>
-        <div class="tab_code_content">
+        <div style="max-height: 450px; overflow-y: scroll;"class="tab_code_content">
```

Comment:
Nit: Should we just set this as fixed instead of max, so that the container size doesn't change at all?

Reply:
Yes - fixed in 67ed3b7